.gitignore

```diff
@@ -20,3 +20,6 @@ bin/
 # iPXE build directories
 upstream-*
 ipxe-*.tar.gz
+
+# added by lint-install
```

Comment:
I mean for deps you use, like `lint-install`

Reply:
its not a big deal really since lint-install isn't called automatically by anything so there's not a huge chance of back and forths but its also nice to get it in early anyway too.